### PR TITLE
A26-18: Redirect backpackers to Map(Dashboard) and all users back to user edit if they have not agreed to privacy policy and terms.

### PR DESCRIPTION
**Pull request by cfblack**

### CODE REVIEW

**Comment by mrkmiller** [comment]
Should this also check for excluded roles on the Agree Subscriber?

**Comment by cfblack** [comment]
@mrkmiller No. Everyone should agree to the privacy policy. The excluded roles was only for people getting redirected to the Map page. Are you talking about a Behat test to see if an excluded role gets redirected?

**Comment by mrkmiller** [comment]
Ok. No, I was wondering if only Backpackers had to accept the privacy policy.

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "drupal/core": "^8",
     "drupal/key": "^1.7",
     "drupal/masquerade": "^2.0@beta",
+    "drupal/me_redirect": "^1.0@beta",
```
Comment (mrkmiller) [inline composer.json]: Is this module actually being used somewhere?

Comment (dearmond) [reply composer.json]: Yes, Carson is using it to redirect /map

Comment (cfblack) [reply composer.json]: yes. It is used in the Backpacker Profile link and will also be used to get to the user account edit screen.

**Pull request merged**